Dockerfile

```diff
@@ -54,7 +54,6 @@ RUN apt update         && \
         curl                  \
         git                   \
         jq                    \
-        realpath              \
```

Comment:
does this mean `realpath` is no longer available there? O.o why not?

Reply:
Just the package not available, the command is in `coreutils` package.

See: https://packages.ubuntu.com/bionic/amd64/coreutils/filelist